instrumentation/spring/spring-webflux/spring-webflux-5.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/spring/webflux/v5_0/server/reactornetty/ReactorNettyInstrumentationModule.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.spring.webflux.v5_0.server.reactornetty;
+
+import com.google.auto.service.AutoService;
+import io.opentelemetry.javaagent.extension.instrumentation.InstrumentationModule;
+import io.opentelemetry.javaagent.extension.instrumentation.TypeInstrumentation;
+import java.util.Arrays;
+import java.util.List;
+
+@AutoService(InstrumentationModule.class)
+public class ReactorNettyInstrumentationModule extends InstrumentationModule {
```

Comment:
Hmm, ideally this should live somewhere in the `:reactor:reactor-netty` directory. WDYT about putting it into `reactor-netty-0.7`? (since that's the lowest reactor version used by webflux)

Reply:
I think we'd need to call it `reactor-netty-server` or `reactor-netty-server-0.7` because the existing `reactor-netty` instrumentations seem to be for http client only. Would that be ok? I mainly chose to put it under webflux because the only tests that exercise this code are in webflux.